Taehoon

aa4758 opened the pull request:
함수

aa4758 committed:
명함인식_테스트

aa4758 committed:
수정완료

aa4758 committed:
자동차 번호판 인식

명함과 다르게 제대로 인식하지 못함

aa4758 committed:
mongodb

견본

aa4758 committed:
mongodb function

함수만 모아놓은 것

With-Coding-Cat commented on mongodb.py:
```diff
@@ -0,0 +1,50 @@
+
+
+def MongoDB(video1, video2, image, time, plate1):
+    db = client['car_database']             # 데이터베이스 불러오기
+    collection = db["collection_1"]         # collection 불러오기
```
위에서는 collection의 이름을 정하게 했는데, 여기서는 애초부터 지정되어 있습니다. 적절히 수정 바랍니다.

With-Coding-Cat replied:
위에서는 데이터베이스와 collection의 이름을 정하게 했는데, 여기서는 애초부터 지정되어 있습니다. 적절히 수정 바랍니다.

With-Coding-Cat commented on mongodb.py:
```diff
@@ -0,0 +1,50 @@
+                  "car license plate":"{}".format(plate1)})     # 번호판
+
+def TrackingDB(plate_input):
+    db = client['car_database']             # 데이터베이스 불러오기
+    collection = db['collection_2']         # collection 불러오기
```
마찬가지로 위에서는 collection의 이름을 정하게 했는데, 여기서는 지정되어 있습니다. 적절히 수정 바랍니다.

With-Coding-Cat replied:
마찬가지로 위에서는 collection의 이름을 정하게 했는데, 여기서는 지정되어 있습니다. 적절히 수정 바랍니다.

With-Coding-Cat commented on mongodb.py:
```diff
@@ -0,0 +1,50 @@
+
+    collection.insert({"car license plate":"{}".format(plate_input)})    # 찾는 번호판 db2에 입력
+    
+    collection = db['collection_1']         # collection 불러오기
```
이름 지정과 관련된 동일한 문제입니다.

With-Coding-Cat commented on mongodb.py:
```diff
@@ -0,0 +1,50 @@
+
+    collection.insert({"car license plate":"{}".format(plate_input)})    # 찾는 번호판 db2에 입력
+    
+    collection = db['collection_1']         # collection 불러오기
+    match_data = db.collection_1.find({"car license plate":"{}".format(plate_input)})    # 찾는 번호판 데이터를 db1에서 호출
```
이름 지정과 관련된 동일한 문제입니다.

With-Coding-Cat commented on mongodb.py:
```diff
@@ -0,0 +1,50 @@
+    match_data = db.collection_1.find({"car license plate":"{}".format(plate_input)})    # 찾는 번호판 데이터를 db1에서 호출
+    
+    if match_data == None:    # 번호판 호출했을 때 없다면 False 있다면 True
+        print('False')
+    else:
+        print("True")
```
return True
return False
형식이 되어야 합니다.

With-Coding-Cat reviewed (changes requested):
.ipynb 형식은 통합하기에 적절하지 않습니다.
필요한 파일이라면 .py로 고쳐주세요.

필요한 부분만 main 브랜치로 옮겨갈 수 있게 파일 전체를 다시 살펴주세요.

그리고 데이터 베이스 관련 부분에 대해 코멘트 해 놓은 곳 살펴서 수정해주세요.
제가 말한 가이드 라인에서 벗어나도 상관은 없지만, 전체적인 통일성이 유지되어야 합니다.

한 함수에서는 데이터 베이스와 collection의 이름을 다 정하도록 되어 있는데, 다른 곳에서는 이미 함수 식에서 미리 정해진 것에만 넣을 수 있게 되어 있습니다.

aa4758 committed:
문제

aa4758 committed:
DB function

Createmongodb() : DB 생성
MongoDB ('abc', 'def', 'ghi', 'jkl', 'mno') : DB에 값 입력
TrackingDB('mno'): 번호판값이 mno인 데이터가 있는지 확인
